pkg/apis/pxc/v1/pxc_backup_types.go

```diff
@@ -65,7 +65,7 @@ type PXCBackupStatus struct {
 	StorageName           string                            `json:"storageName,omitempty"`
 	S3                    *BackupStorageS3Spec              `json:"s3,omitempty"`
 	Azure                 *BackupStorageAzureSpec           `json:"azure,omitempty"`
-	FsPvc                 *corev1.PersistentVolumeClaimSpec `json:"fsPvc,omitempty"`
+	Pvc                   *corev1.PersistentVolumeClaimSpec `json:"pvc,omitempty"`
```

Comment:
acronyms should be all upper case

https://go.dev/wiki/CodeReviewComments#initialisms